install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml

```diff
@@ -41,4 +41,7 @@ rewriteAppHTTPProbe: false
 # See https://istio.io/docs/setup/kubernetes/additional-setup/sidecar-injection/#more-control-adding-exceptions
 neverInjectSelector: []
 
-alwaysInjectSelector: []
\ No newline at end of file
+alwaysInjectSelector: []
+
+# Enable managing webhookconfiguration
```

Comment:
can you expand on this - what does manage mean? what is webhook configuration? why should I set this?

I think I know, but these docs are user facing and the average user may not

Reply:
Done: the explanation of this parameter has been expanded.